src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -30,24 +24,22 @@ class JsonAdaptedPerson {
     private final String email;
     private final String address;
     private final String remark;
-    private final List<JsonAdaptedTag> tagged = new ArrayList<>();
+    private final JsonAdaptedTag tagged;
 
     /**
      * Constructs a {@code JsonAdaptedPerson} with the given person details.
      */
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
                              @JsonProperty("email") String email, @JsonProperty("address") String address,
-                             @JsonProperty("tagged") List<JsonAdaptedTag> tagged,
+                             @JsonProperty("tagged") JsonAdaptedTag tagged,
```

Comment:
Perhaps we could replace the type here with just string and not `JsonAdaptedTag`.